assembl/static2/js/app/utils/routeMap.js

```diff
@@ -15,7 +15,14 @@ const convertToContextualName = (name) => {
   return base + workingName;
 };
 const maybePrependSlash = (pre, s) => (pre ? `/${s}` : s);
-export const get = (name, args) => {
+
+export const getQuery = (query) => {
+  if (!query) return '';
+  const entries = Object.keys(query).map(key => `${key}=${query[key]}`);
+  return `?${entries.join('&')}`;
+};
```

Comment:
We should maybe start using `query-string` dependency (we already have query-string in the project via react-router/history dependency, but you should add it explicitly in package.json) instead of writing our own function here.

remove this getQuery function and below in `get` function use:
```
import { stringify } from "query-string";

return `${a}?${stringify(query)}`
```